src/transport/PairingSession.h

```diff
@@ -36,6 +36,28 @@ class DLL_EXPORT PairingSession
     PairingSession() {}
     virtual ~PairingSession() {}
 
+    NodeId GetPeerNodeId() const { return mPeerNodeId; }
+    void SetPeerNodeId(NodeId peerNodeId) { mPeerNodeId = peerNodeId; }
+
+    uint16_t GetPeerKeyId() const { return mPeerKeyId; }
+    void SetPeerKeyId(uint16_t id) { mPeerKeyId = id; }
+    bool IsValidPeerKeyId() const { return mPeerKeyId != kInvalidKeyId; }
```

Comment:
Looks unused... But also, if GetPeerKeyId returned `Optional` this is not needed?

Reply:
Replaced with optional